src/audio/pipeline.c

```diff
@@ -972,10 +972,10 @@ static uint64_t pipeline_task(void *arg)
 			trace_pipe_error_with_ids(p, "pipeline_task(): xrun "
 						  "recover failed! pipeline "
 						  "will be stopped!");
-			return 0; /* failed - host will stop this pipeline */
+			goto end;
 		}
 	}
-
-	tracev_pipe("pipeline_task() sched");
-	return p->ipc_pipe.period;
+end:
```

Comment:
Maybe `out` would be better for consistency with other labels like that?

Reply:
No longer needed.